op-bindings/predeploys/addresses.go

```diff
@@ -34,6 +34,7 @@ const (
 	UniswapFeeHandlerSeller           = "0xd3aee28548dbb65df03981f0dc0713bfcbd10a97"
 	SortedOracles                     = "0xefb84935239dacdecf7c5ba76d8de40b077b7b33"
 	AddressSortedLinkedListWithMedian = "0xED477A99035d0c1e11369F1D7A4e587893cc002B"
+	FeeCurrency                       = "0x4200000000000000000000000000000000000022"
```

Comment:
Should we leave some room in the address space for upstream to add more contracts? That could reduce diff size and merge conflicts in the future.

Reply:
That is very good point, I added 1000 to the address